src/typings/prop-types.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'prop-types' {
+    import React = require('react');
+
+    export const bool: {
+        isRequired: React.Validator<any>
```

Comment:
I tried to npm install **@types/prop-types** but it won't compile because of error TS2312, that's why I modified the code to convert the interface extending the type for an implementation of both the interface and the type on each property.

If I remove all references to React.PropTypes in react.d.ts, I suspect that the references in Image.tsx, Text.tsx and View.tsx won't compile, am I wrong?